src/configs/recommended.ts

```diff
@@ -56,5 +57,14 @@ export function recommended(extenions?: RuleExtensions): FlatConfig.ConfigArray
         ...tsRules(extenions?.ts),
       },
     },
+    {
+      plugins: {
+        import: pluginImport,
+      },
+      rules: {
+        // https://github.com/un-ts/eslint-plugin-import-x/blob/master/docs/rules/order.md
+        'import/order': ['error', { alphabetize: { order: 'asc' } }],
```

Comment:
Can we double check if it works correctly that it doesn't re-order imports that are separated by newlines? So that each block of imports is independently sorted and not all imports are sorted / grouped? There is often the use case to have say `.css` imports in a separate block to group them a bit by JS and styles for instance.

Reply:
Very cool!

Reply:
The only thing would be that I can prolly no longer use the "organize imports" in VSC since it doesn't match the ordering of this plugin. That's very unfortunate cause it's so easy but I guess the plugin provides a fixer?

Reply:
Yup, there's autofix! Just run ESLint autofixes on save and it provides similar experience.

Reply:
Okay, I guess I can re-map maybe that shortcut to run eslint or something. I have to look into it unless you know if that's possible.

Reply:
I've used just ESLint and prettier plugins for sorting the imports. IDE's modifying code is not consistent across all developers so I try to avoid those. 😄